core/server/data/db/connection.js

```diff
@@ -26,6 +26,22 @@ function configure(dbConfig) {
         dbConfig.useNullAsDefault = dbConfig.useNullAsDefault || false;
     }
 
+    if (client === 'mysql') {
+        dbConfig.timezone = 'UTC';
+    }
+
+    // https://github.com/tgriesser/knex/issues/97
+    // this sets the timezone to UTC only for the connection!
+    if (client === 'pg') {
```

Comment:
I would move the code up inside the other PG block? 

Might make sense to have two separate functions though - one for sorting out the strings, and one for timezones.